eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/DelegSpec.hs

```diff
@@ -476,6 +478,77 @@ spec = do
       expectNotRegistered cred
       expectNotDelegatedVote cred
 
+  describe "Delegate both stake and vote - separated out for conformance mismatch" $
+    -- https://github.com/IntersectMBO/formal-ledger-specifications/issues/640
+    -- TODO: Re-enable after issue is resolved, by removing this override
+    modifyImpInitExpectLedgerRuleConformance (\_ _ _ _ _ -> pure ()) $ do
```

Comment:
This override could be useful elsewhere so maybe it should be its own function e.g. `disableConformance`. That way we can also easily grep for the function name once we start re-enabling the tests.